### PR TITLE
Data details with smaller render

### PR DESCRIPTION
**Pull request by cschroed-usgs**

### CODE REVIEW

File: src/main/webapp/js/advanced/views/DataDetailsView.js [hunk 1]
```diff
@@ -89,36 +116,57 @@ var GDP = GDP || {};
...
+	 * @param {GDP.ADVANCED.model.JobModel} 
+	 * @param {String} url 
 	 * @returns {jQuery.Deferred.promise} The promise is resolved with no args 
 	 * if user cleared the url or if user submitted a url and all subesequent 
 	 * web service calls succeded. The promise is rejected with an error message
 	 * if any web service calls fail, or if the web service responses cannot be
 	 * parsed.
 	 */
-	'changeUrl': function (ev) {
+	'changeUrl': function (jobModel, url) {
```
Comment (mbucknell) [inline src/main/webapp/js/advanced/views/DataDetailsView.js]: It's a little inconsistent to specify callback arguments here but not in any of the other of the model event handlers.

File: src/main/webapp/js/util/SelectMenuView.js [hunk 2]
```diff
@@ -24,23 +21,28 @@ GDP.util.SelectMenuView = Backbone.View.extend({
 		"use strict";
 
 		this.emptyPlaceholder = options.emptyPlaceholder ? options.emptyPlaceholder : false;
-		this.sortOptions = options.sortOptions ? options.sortOptions : false;
+		
+		//'text', 'value', 'selected', or a custom function
+		this.sortBy = options.sortBy ? options.sortBy : false;
+		
+		//true for ascending, false for descending
+		this.sortAscending = _.isUndefined(options.sortAscending) ? true : options.sortAscending;
```
Comment (mbucknell) [inline src/main/webapp/js/util/SelectMenuView.js]: Like these changes!

File: src/main/webapp/js/advanced/views/SpatialView.js [hunk 3]
```diff
@@ -288,16 +296,20 @@ GDP.ADVANCED.view = GDP.ADVANCED.view || {};
...
-							// Don't repeat values in the list
-							var value = $(this).text();
-							if (_.indexOf(optionValues, value) === -1) {
-								optionValues.push(value);
-							}
+						// Don't repeat values in the list
+						var optionValues = _.uniq(
+							_.map($(data).find(attribute), function(datum) {
+								return $(datum).text();
+							})
```
Comment (mbucknell) [inline src/main/webapp/js/advanced/views/SpatialView.js]: I've got to start looking in _underscore for more of this syntatic sugar!

**Comment by mbucknell** [comment]
Other than the changeUrl comment, I think this is good to go :+1: .

File: src/main/webapp/js/advanced/views/DataDetailsView.js (hunk 1 above)
Comment (cschroed-usgs) [inline src/main/webapp/js/advanced/views/DataDetailsView.js]: Good point. I'll switch it to ask the model for the current url property.

File: src/main/webapp/js/advanced/views/SpatialView.js (hunk 3 above)
Comment (cschroed-usgs) [inline src/main/webapp/js/advanced/views/SpatialView.js]: Underscore and sugarjs definitely make life easier.

File: src/main/webapp/js/util/SelectMenuView.js (hunk 2 above)
Comment (cschroed-usgs) [inline src/main/webapp/js/util/SelectMenuView.js]: Thanks!

**Comment by cschroed-usgs** [comment]
Thanks for the feedback. I changed changeUrl. Mergin!

**Pull request merged**